java/cdk/src/main/java/sleeper/cdk/stack/DashboardStack.java

```diff
@@ -254,7 +254,7 @@ private void addTableWidgets() {
                             .title("FilesPerPartition")
                             .left(Collections.singletonList(Metric.Builder.create()
                                     .namespace(metricsNamespace)
-                                    .metricName("AverageActiveFilesPerPartition")
+                                    .metricName("")
```

Comment:
This needs filling in, and the title needs changing.

Reply:
Comment addressed